tests/test_app_gallery.py

```diff
@@ -201,7 +201,7 @@ def validate_app_functionalities(app_page: "Page") -> None:
         ".MuiGrid-container"
     )
     search_results_container.wait_for(timeout=150 * 1000)
-    sleep(5)
+    sleep(10)
```

Comment:
@krshrimali 

if we want to wait longer until the result appear, it should be the previous line: 
`search_results_container.wait_for(timeout=150 * 1000)`

we wait 150 seconds now, maybe to increase it a little bit only to 180 ?

Reply:
Ah yes, let me do this.

Reply:
Done!!